Rakefile.rb

```diff
@@ -23,9 +29,12 @@
     sh "cd #{PROJECT_DIR} && bundle update"
     puts "building integration #{ENV['INTEGRATION']}"
 
+    manifest = JSON.parse(File.read("/#{ENV['INTEGRATIONS_REPO']}/#{ENV['INTEGRATION']}/manifest.json"))
+    integration_version = manifest['version'] || ENV['VERSION']
```

Comment:
If there's a divergence, what does that mean ? What's right ? Or is it only for `ENV['VERSION'] == 'master'` ?